postman/sanity.postman_collection.json

```diff
@@ -0,0 +1,89 @@
+{
+	"info": {
+		"_postman_id": "4a4deeb8-dae7-4bdc-98ba-c457b1bffff1",
+		"name": "sanity",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Submit Claim",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+						"mode": "formdata",
+						"formdata": [
+								{
+										"key": "file",
+										"type": "file",
+										"enabled": true,
+										"src": "claim.cose"
+								}
+						]
+				},
+				"url": {
+					"raw": "http://127.0.0.1:8000/entries",
```

Comment:
Looks like a lot of duplication here? Is there a short form for "url"?

Reply:
you can use variable names, that is probably a thing to do, after the basic tests are running in CI.

Reply:
deferring for now, these will become parameters later.